Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM node:18-alpine AS base
+
+WORKDIR /app
+
+# ----------------------------------------
+# Stage 1: Prepare package.json files
+FROM base AS installer
+
+# Install turbo
+RUN npm install -g turbo@1.13.3
+
+# COPY the whole project to the container
+COPY . .
```

Comment:
maybe there is a way to just ignore this line for sonarcube

Reply:
Looks like it has to be done from their UI: https://community.sonarsource.com/t/nosonar-in-docker-file/95235

Reply:
Could you please clarify what to do with the first link? I don't understand how the specified function can be used in our case

Reply:
```suggestion
COPY . . // NOSONAR
```

Reply:
Unfortunately the NOSONAR didn't work, I have tried multiple syntax versions of it.
Going now with the third method discussed in our meeting, adding a comment for clarification and manually approving it in SonarQube UI.
@derberg kindly check